mandatory/2-function-creation.js

```diff
@@ -6,7 +6,13 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num === "number" && num / 2 === 0 && num <= 100 ){
+    return true; 
+  } else {
+    return false;
+  }
+}
```

Comment:
Great approach here too! There's a typo at line10 : num / 2 === 0 instead of num % 2 === 0 , so the if condition will always be false  :)

Reply:
Nice spot, thanks Maha